tests/fixtures/sample_lib/modules/sample_task/list_implementation.py

```diff
@@ -34,7 +34,10 @@ def load(cls, model_path, **kwargs):
         return cls(config["train"]["batch_size"], config["train"]["learning_rate"])
 
     def run(
-        self, sample_input: SampleInputType, throw: bool = False
+        self,
+        sample_input: SampleInputType,
+        throw: bool = False,
+        error_obj: Optional[Exception] = None,
```

Comment:
unused? but if intended to allow for future use maybe it'd be helpful to keep consistent as either exception or str between `sample_implementation` and this?

Reply:
accidentally pushed! Removing